pa11ycrawler/cmdline.py

```diff
@@ -198,6 +204,10 @@ def main():
     parser = cli()
     args = parser.parse_args()
     settings = args_to_settings(args)
+    logging.basicConfig(
+        format='%(asctime)s [%(levelname)s] [%(name)s]: %(message)s',
+        level=settings.get('LOG_LEVEL')
```

Comment:
I don't see a default value here. I'm assuming this reads from environment variables? What happens if it's not set?